From e8498bb1ab3368314aab54f3d55c991edc469062 Mon Sep 17 00:00:00 2001
From: qwang2 <quanyang.wang@windriver.com>
Date: Tue, 3 Apr 2018 17:16:59 +0800
Subject: [PATCH] mtd: spi-nor: fix SPI_S3AN and SPI_NOR_DDR_QUAD_READ flag bit
 conflict

In i.mx SDK, SPI_NOR_DDR_QUAD_READ has been set as BIT(10). But this
is conflict with SPI_S3AN which introduced from upstream commit.
Move SPI_NOR_DDR_QUAD_READ from BIT(10) to BIT(13).

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mtd/spi-nor/spi-nor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/spi-nor/spi-nor.c b/drivers/mtd/spi-nor/spi-nor.c
index 5f1b0f2..a9c8e78 100644
--- a/drivers/mtd/spi-nor/spi-nor.c
+++ b/drivers/mtd/spi-nor/spi-nor.c
@@ -76,7 +76,6 @@ struct flash_info {
 					 * bit. Must be used with
 					 * SPI_NOR_HAS_LOCK.
 					 */
-#define	SPI_NOR_DDR_QUAD_READ	BIT(10)  /* Flash supports DDR Quad Read */
 #define	SPI_S3AN		BIT(10)	/*
 					 * Xilinx Spartan 3AN In-System Flash
 					 * (MFR cannot be used for probing
@@ -88,6 +87,7 @@ struct flash_info {
 					 * to support memory size above 128Mib.
 					 */
 #define NO_CHIP_ERASE		BIT(12) /* Chip does not support chip erase */
+#define	SPI_NOR_DDR_QUAD_READ	BIT(13)  /* Flash supports DDR Quad Read */
 };
 
 #define JEDEC_MFR(info)	((info)->id[0])
-- 
1.8.5.2.233.g932f7e4

